### ISSUE TITLE
React 17 support

### ISSUE THREAD

**Issue by javlund** [sarink/react-file-drop#81]
This lib seems to have a peer dependency on React 16, which with the current version, doesn't fly with the newest React:

```
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.13.1" from react-file-drop@3.1.2
npm ERR! node_modules/react-file-drop
npm ERR!   react-file-drop@"3.1.2" from the root project
```

Is the library still updated, and if so, are there any plans to support React 17?

**Comment by javlund** [comment]
Ah - now I see https://github.com/sarink/react-file-drop/pull/78. Seems like the lib is abandoned then. :/

**Comment by sarink** [comment]
Released in v3.1.3
